test/system/800-system-reset.bats

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# tests for podman system df
+#
+
+load helpers
+
+function setup() {
+    if is_remote; then
+        skip "only applicable when running non remote"
+    fi
+
+    basic_setup
+}
+
+
+function teardown() {
+    basic_teardown
+
+    # In case the active-volumes test failed: clean up stray volumes
+    run_podman volume rm -a
+}
+
+
+@test "podman system reset - storage options" {
+    whoami=$(id -un)
+    if ! is_rootless; then
+        run_podman system reset -f --user=$whoami
+        if [ "$status" -ne 0 ]; then
```

Comment:
`run_podman` does implicit exit-status checks: you do not (and should not) need to check `$status`. (With a very narrow exception that does not apply here).

What you _should_ always do is check `$output`.

Reply:
ok thanks, that makes more sense.Two questions on these tests: 1) is it ok to change the run root and graph root here or will it change the entire system's? and 2) I was unsure of what to name the file is `800-` an ok number to use?

Reply:
Changing `--root` and `--runroot` (no hyphen) is always (AFAIK) safe. There are a handful of cases where we already do that in the test. `graphroot`, I have no idea: this is the first I've ever heard of that option, and I don't even see it in `--help`.

`800` is a great choice for something as scary as `system reset`.

Reply:
Of course. `grep 'is.*\.\*' test/system/*bats`